Vagrantfile

```diff
@@ -171,7 +182,7 @@ scissor = {
     :autostart => true,
     :box => CENTOS_BOX_ID,
     :cpus => 1,
-    :mac_address => "0800271F9D12",
+    :mac_address => "0800271F9D13",
```

Comment:
Don't change MAC addresses. They are used to statically assign IP addresses via DHCP.